README.md

```diff
@@ -1,13 +1,38 @@
 [![Build Status](https://travis-ci.org/ofiwg/libfabric.svg?branch=master)](https://travis-ci.org/ofiwg/libfabric)
 
-fabtests
-========
+# fabtests
 
-Fabtests provides a set of examples that uses [libfabric](https://github.com/ofiwg/libfabric) - a high-performance fabric software library.
+Fabtests provides a set of examples that uses
+[libfabric](https://github.com/ofiwg/libfabric) -a high-performance fabric
+software library.
 
-To make this directory, run:
+## Building fabtests
 
-    ./autogen.sh && ./configure && make && make install
+Fabtests uses GNU autotools. A typical install looks like:
+
+```
+./autogen.sh && ./configure && make && make install
+```
 
 Typically the autogen and configure steps only need be done the first
 time, unless configure.ac changes.
+
+Configure ships with many built in options (see `./configure -h`). Some useful
+options are:
+
+```
+--prefix=<directory>
+  By default "make install" will place the files into a system wide location.
+  The prefix option specifies that the fabtests files should be installed into
+  the directory named <directory>. The executables will be located at
+  <directory>/bin.
+
+--with-libfabric=<directory>
+  Specify the directory where the libfabric library and header files are
+  located. This is necessary if libfabric was installed in a non-standard
+  location and the files are not present in the compiler/linker search paths.
```

Comment:
Maybe add an example like what was done for prefix?  Files are searched for in <directory>/lib ...